doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1458 @@
+  * Configuration options and SUIT manifests.
+
+* Fixed:
+
+  * An issue where usage of ``nrfutil device recover`` leads to setting SUIT recovery (NCSDK-29465).
+  * An issue where envelopes with SUIT components not needed for particular roles are not rejected as candidates (NCSDK-29237).
+  * An issue where a duplicate class ID in MPI leads to cyclic SDFW resets (NCSDK-28377).
+  * An issue where leftover data in SUIT storage prevents writing to SUIT storage for a new application or radio image (NCSDK-28317).
+  * An issue where ``smp_transfer`` west flash fails with cyclic resets afterwards (NCSDK-27732).
+  * An issue where the **suit-generator** generation of installed manifest images for custom manifest class ID is not possible (NCSDK-27289).
+  * An issue where manifest could declare any ``INSTLD_MFST`` component regardless of it's class ID (NCSDK-27235).
+
+Developing with nRF91 Series
+============================
+
+* Added:
+
+  * User guides for Thingy:91 X in the :ref:`ug_nrf91` documentation.
+  * The :ref:`nRF91 modem tracing with RTT backend snippet <nrf91_modem_trace_rtt_snippet>` to enable modem tracing using the RTT trace backend.
+  * The :ref:`nRF91 modem tracing with RAM backend snippet <nrf91_modem_trace_ram_snippet>` to enable modem tracing using the RAM trace backend.
+
+Developing with nRF70 Series
+============================
+
+* The :ref:`ug_nrf70_features_hostap` section to the :ref:` ug_nrf70_features` page
+
+Developing with nRF54L Series
+=============================
+
+* Added:
+
+  * :ref:`nRF54l_snippets` to emulate these targets on an nRF54L15 DK.
+    These are used only for development purposes.
+  * The :ref:`ug_nrf54l_cryptography` page that provides more information about the cryptographic peripherals of the nRF54L Series devices, programming model for referencing keys, and configuration.
+  * A page on :ref:`memory_storage` system.
+  * The :ref:`vpr_flpr_nrf54l` and :ref:`building_nrf54l` pages.
+
+* Updated the name and the structure of the section, with :ref:`ug_nrf54l` as the landing page.
+* Removed the Getting started with the nRF54L15 PDK page, and instead included the information about the `Quick Start`_ app support.
+
+Security
+========
+
+* Added:
+
+  * The :kconfig:option:`CONFIG_CRACEN_IKG_SEED_KMU_SLOT` Kconfig option to allow customization of the KMU slot used to store CRACEN's Internal Key Generator (IKG) seed.
+    The default IKG seed slot is now 183 (previously 0).
+  * TF-M support to the :ref:`zephyr:nrf54l15dk_nrf54l15` (board target ``nrf54l15dk/nrf54l15/cpuapp/ns``), replacing the :ref:`zephyr:nrf54l15pdk_nrf54l15` (board target ``nrf54l15pdk/nrf54l15/cpuapp/ns``).
+
+* Deprecated legacy Mbed TLS crypto toolbox APIs that are enabled when the :kconfig:option:`CONFIG_NORDIC_SECURITY_BACKEND` Kconfig option is set.
+  Use the nRF Security (enabled with the :kconfig:option:`CONFIG_NRF_SECURITY` Kconfig option) and PSA crypto APIs instead.
+
```

Comment:
* Added `west ncs-provision` command which allows to provision signature verification keys to the nRF54l SoS over J-Link interface. See